examples/chat/chat_server/lib/src/generated/channel.dart

```diff
@@ -24,15 +24,11 @@ abstract class Channel extends _i1.TableRow {
     required String channel,
   }) = _ChannelImpl;
 
-  factory Channel.fromJson(
-    Map<String, dynamic> jsonSerialization,
-    _i1.SerializationManager serializationManager,
-  ) {
+  factory Channel.fromJson(Map<String, dynamic> jsonSerialization) {
     return Channel(
-      id: serializationManager.deserialize<int?>(jsonSerialization['id']),
-      name: serializationManager.deserialize<String>(jsonSerialization['name']),
-      channel: serializationManager
-          .deserialize<String>(jsonSerialization['channel']),
+      id: jsonSerialization['id'] as int?,
+      name: jsonSerialization['name'] as String,
```

Comment:
Yes, it will throw these exceptions, but you will be able to determine which field is causing the exception using the stack trace. This makes debugging clear for developers during development. 
Such errors should never be visible to users or clients using your app/platform. 
So in my opinion for debugging this error its good enough, if you have any suggestion let me know.